python/cudf/cudf/_lib/csv.pyx

```diff
@@ -101,7 +101,7 @@ cdef csv_reader_options make_csv_reader_options(
     bool na_filter,
     object prefix,
     object index_col,
-) except +:
+) except *:
```

Comment:
I was getting a segfault, so I changed this. C++ isn't the only code that can throw an exception in this function.

Reply:
This looks like it should work fine to me!